mercadopago/px-ios
# (Added)[PXN-4162] Added notifications to communicate between PX and Integrator

## What has changed?

Added _PX -> Integrator_ notifications, to inform on certain events occurring and allowing the integrator to optionally call a method on each of them.

## How to test:

- Add a builder integration with an Advanced Configuration, setting _CustomReviewAndConfirmViewController_ as the injected VC. 
- Check that the processes are getting called in console.
